disperser/dataapi/v2/batches.go

```diff
@@ -0,0 +1,148 @@
+package v2
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/disperser/dataapi"
+	"github.com/gin-gonic/gin"
+)
+
+// FetchBatchFeed godoc
+//
+//	@Summary	Fetch batch feed
+//	@Tags		Batches
+//	@Produce	json
+//	@Param		end			query		string	false	"Fetch batches up to the end time (ISO 8601 format: 2006-01-02T15:04:05Z) [default: now]"
+//	@Param		interval	query		int		false	"Fetch batches starting from an interval (in seconds) before the end time [default: 3600]"
+//	@Param		limit		query		int		false	"The maximum number of batches to fetch. System max (1000) if limit <= 0 [default: 20; max: 1000]"
+//	@Success	200			{object}	BatchFeedResponse
+//	@Failure	400			{object}	ErrorResponse	"error: Bad request"
+//	@Failure	404			{object}	ErrorResponse	"error: Not found"
+//	@Failure	500			{object}	ErrorResponse	"error: Server error"
+//	@Router		/batches/feed [get]
+func (s *ServerV2) FetchBatchFeed(c *gin.Context) {
+	handlerStart := time.Now()
```

Comment:
Since these are handlers could we put them all under a `handlers` directory or do you prefer keeping it at top level.

Reply:
Yea I considered it, just feel the nesting is quite deep already. We may see if there are more other files coming up under `v2`, and if so it may make more sense to group these handlers in under `handlers`.

Reply:
AI's comments:
![dataapi handlers](https://github.com/user-attachments/assets/0d50dcac-e1d4-41ee-bbca-307b51eb4e5d)

Reply:
and:
![grouping](https://github.com/user-attachments/assets/0202a718-be20-46b7-90eb-d55498531d4f)